multi-node/aws/pkg/config/config_test.go

```diff
@@ -77,6 +82,8 @@ controllerIP: 10.4.3.5
 podCIDR: 172.4.0.0/16
 serviceCIDR: 172.5.0.0/16
 dnsServiceIP: 172.6.100.101 #dnsServiceIP not in service CIDR
+`, `
+routeTableId: rtb-xxxxxx # routeTableId specified without vpcId
```

Comment:
Given that you're not testing the actual failure type, this test is likely failing for other reasons (has no other config items). Ideally we could also test for the expected failure, but in the short term you could at least add the rest of the values to make this an otherwise passing case (with the exception of the missing vpcID).